app/code/Transoft/Blog/Block/Singleblog.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
Why do we need this block?

Reply:
This block is used to render blog on a single page.

Reply:
Please use Block or ViewModel, but not both.

Reply:
fixed, removed ViewModel